echo/server/dembrane/transcribe.py

```
@@ -201,43 +202,79 @@ def transcribe_conversation_chunk(conversation_chunk_id: str) -> str | None:
-                return None
+            try:
+                response = requests.get(runpod_job_status_link, headers=headers, timeout=30)
+                response.raise_for_status()  # Raise an exception for bad status codes
+
+                response_data = response.json()
+                logger.debug(f"RunPod status response: {response_data}")
+
+                job_status = response_data.get("status")
+                if job_status is None:
+                    logger.warning(
+                        f"No 'status' field in RunPod response for {runpod_job_status_link}: {response_data}"
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)